projects/js-packages/publicize-components/src/components/media-section/index.js

```diff
@@ -110,14 +137,41 @@ export default function MediaSection() {
 						<MediaUpload
 							title={ REPLACE_MEDIA_LABEL }
 							onSelect={ onUpdateMedia }
-							allowedTypes={ ALLOWED_MEDIA_TYPES }
+							allowedTypes={ allowedImageTypes }
 							render={ replaceMediaRender }
 						/>
 						<Button onClick={ onRemoveMedia } variant="link" isDestructive>
 							{ REMOVE_MEDIA_LABEL }
 						</Button>
 					</>
 				) }
+				{ validationError && (
+					<Notice
+						className={ styles.notice }
+						isDismissible={ true }
+						onDismiss={ onDismissClick }
+						status="warning"
+					>
+						{ validationError === 1 && (
+							<p>{ __( 'The selected media type not accepted by these platforms.', 'jetpack' ) }</p>
```

Comment:
Bit of a nitpick, but I think this is missing an 'is'
```suggestion
							<p>{ __( 'The selected media type is not accepted by these platforms.', 'jetpack' ) }</p>
```